scripts/run-generate-subgraph-types.sh

```diff
@@ -6,4 +6,4 @@ else
   export SUBGRAPH_GRAPHQL_ENDPOINT="https://api.thegraph.com/subgraphs/name/venusprotocol/venus-isolated-pools-chapel"
 fi
 
-yarn graphclient build --dir ./src/types/subgraph/
+yarn graphql-code-generator --config ./src/types/subgraph/codegen.ts
```

Comment:
Because we now need to access `SUBGRAPH_GRAPHQL_ENDPOINT` from within the app in order to call the subgraph using `graphql-request`, I'd make it available from the config directly (which you can find at `src/config/index.ts`) using the same logic as for the API and RPC URLs.

  This way it can then be imported from `codegen.ts` using the config directly (and doesn't need to be set in the `.env` file, since it's anyway static).
  
  This means this script file can be removed, and instead the command `yarn graphql-code-generator --config ./src/types/subgraph/codegen.ts` can be used directly inside the `package.json` file without needing to set any environment variable.